models/sipnet/R/read.restart.SIPNET.R

```diff
@@ -34,17 +33,18 @@ read.restart.SIPNET <- function(outdir,runid,time,settings,variables,sample_para
     last = length(ens$NPP)
     
     forecast<-numeric(8)
-  
-    forecast[1] <- mean(ens$NPP)*unit.conv ## kg C m-2 s-1 -> Mg/ha/yr [Check]
-    forecast[2] = ens$AbvGrndWood[last]*1000 ## kgC/m2 -> gC/m2
-    forecast[3] = ens$LeafC[last]*prior.sla*2 ## kgC/m2*m2/kg*2kg/kgC -> m2/m2
-    forecast[4] = ens$Litter[last]*1000 ##kgC/m2 -> gC/m2
-    forecast[5] = ens$TotSoilCarb[last]*1000 ## kgC/m2 -> gC/m2
-    forecast[6] = ens$SoilMoistFrac[last] ## unitless
-    forecast[7] = ens$SWE[last]*0.1 ## kg/m2 -> cm
     
-    forecast[8] = runif(1,0,0.01) #snow
-    #forecast$microbe = NA
+    unit.conv <- (10000/1)*(1/10000)*(365.25*24*60*60)
+  
+    #### PEcAn Standard Outputs
+    forecast[1] <- mean(ens$NPP) * unit.conv ## kgC m-2 s-1 -> MgC/ha/yr
```

Comment:
@araiho Is the note here right? I thought the standard units were already kgC/m2/s http://nacp.ornl.gov/MsTMIP_variables.shtml

At this point, if the units here and below are internally consistent, lets not change them again (no more bugs!). But double check that everything is documented correctly and then next release we can update the units.